deploy/gke-marketplace-app/server-deployer/chart/triton/Chart.yaml

```diff
@@ -28,4 +28,4 @@ apiVersion: v1
 appVersion: "2.53"
```

Comment:
suppose this version relevant to `version:` value

Reply:
updated.